scripts/dev/dockerfile_generator.py

```diff
@@ -1,59 +1,93 @@
 import jinja2
 import argparse
+import os
 
 
-def operator_params():
+def operator_params(files_to_add):
```

Comment:
we need type hints on parameters
```suggestion
def operator_params(files_to_add: List[str]):
```

Reply:
Thanks! This was removed due to issue with the Python version on evg and I forgot to to put them back when switching to virtualenv!